live-examples/js-examples/intl/meta.json

```diff
@@ -60,6 +60,12 @@
             "title": "JavaScript Demo: Intl.ListFormat",
             "type": "js"
         },
+        "intlLocalePrototypeToString": {
+            "exampleCode": "./live-examples/js-examples/intl/intl-locale-prototype-tostring.html",
+            "fileName": "intl-locale-prototype-tostring.html",
+            "title": "JavaScript Demo: Intl.Locale.prototype.toString",
```

Comment:
As for the others, this should be "JavaScript Demo: Intl.Locale.prototype.toString()"

Reply:
Changes made! :D